zlib-ng.h.in

```diff
@@ -1792,26 +1787,6 @@ uint32_t zng_crc32_combine_op(uint32_t crc1, uint32_t crc2, const uint32_t *op);
 
                         /* various hacks, don't look :) */
```

Comment:
Do we need the comment above?

Reply:
I think that comment goes for the entire rest of the file, at least that is my take on it, that no outside programmer needs to look below that line.